DAL/pojos/challengeEntity.js

```diff
@@ -0,0 +1,118 @@
+module.exports = class challenge  {
+    challengeId;
+    userId;
+    tittle;
+    description;
+    followers;
+    startDate;
+    endDate;
+    days;
+    timestamp;
+
+   
+    
+    constructor(
+    challengeId,
+    userId,
+    tittle,
+    description,
+    followers,
+    startDate,
+    endDate,
+    days,
+    timestamp,
```

Comment:
as the above.
timestamp could be undefined value.
Try this instead:
timestamp = undefined,